plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/frontend.ts

```diff
@@ -248,6 +275,13 @@ const { actions, state } = store<
 				} else {
 					actions.validateQuantity( productId, value );
 				}
+
+				if ( changeTarget ) {
+					dispatchChangeEvent( changeTarget );
+				}
+
+				// Clear the draft quantity.
+				state.draftQuantity = undefined;
```

Comment:
That's personal opinion, but I believe that using `null` here would be more explicit to make it clear that the variable is intentionally empty. For example, an extension accessing `state.draftQuantity` won't know whether they are using the right variable or not if they find it's `undefined`, while if it's `null` they know it's the correct variable, but it's currently empty.

But as mentioned, that's very subjective. 😄